liteclient/client.go

```diff
@@ -108,14 +138,29 @@ func PrepareTransaction(inputsBody string, outputsBody string) string {
 		newTransaction.PushOutput(addr, out.Coins, out.Hours)
 	}
 
-	newTransaction.SignInputs(keys)
+	if signatureList == nil {
+		newTransaction.SignInputs(keys)
+	} else {
+		newTransaction.Sigs = make([]cipher.Sig, len(signatureList))
+		for i, sig := range signatureList {
+			decodedSig, e := hex.DecodeString(sig)
```

Comment:
cipher.MustSigFromHex can be used

Reply:
You're right, that way the code is much cleaner. Change made